### ISSUE TITLE
Compilation on Linux broken as of c21467e

### ISSUE THREAD

**Issue by naclander** [dBiz/VCVRack_Modules#14]
When compiling on Linux on current master, I get a compilation error showing:

```sh
g++ -fPIC -I../../include -I../../dep/include -MMD -g -O3 -march=nocona -ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_LIN -Wsuggest-override -std=c++11 -c -o build/src/PerfMixer.cpp.o src/PerfMixer.cpp
src/PerfMixer.cpp:10:10: fatal error: dsp/VUMeter.hpp: No such file or directory
 #include "dsp/VUMeter.hpp"
          ^~~~~~~~~~~~~~~~~
compilation terminated.
../../compile.mk:53: recipe for target 'build/src/PerfMixer.cpp.o' failed
make: *** [build/src/PerfMixer.cpp.o] Error 1
```

Did you introduce any new dependencies? The project compiles fine on the previous commit.

By the way, thanks for the great plugin.

**Comment by davephillips** [comment]
The fix: Change VUMeter.hpp to vumeter.hpp

**Comment by naclander** [comment]
@davephillips thanks, I think the fix is to merge #15
